From 9a9cfd9eadac06e7c8040be30930f6c9c6e1c3e8 Mon Sep 17 00:00:00 2001
From: tarsin <43900799+asuka-mio@users.noreply.github.com>
Date: Mon, 15 Aug 2022 13:38:08 +0800
Subject: [PATCH] Add missing include to fix 6.0-rc1 build

https://github.com/torvalds/linux/commit/720cf96d8fecde29b72e1101f8a567a0ce99594f
Upstream removed drm_framebuffer.h from drm_crtc.h which cause drm_framebuffer_get function not found in dkms build
---
 kernel/nvidia-drm/nvidia-drm-helper.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/nvidia-drm/nvidia-drm-helper.h b/kernel/nvidia-drm/nvidia-drm-helper.h
index ecc5ecfdb..455c96a70 100644
--- a/kernel/nvidia-drm/nvidia-drm-helper.h
+++ b/kernel/nvidia-drm/nvidia-drm-helper.h
@@ -108,6 +108,7 @@ nv_drm_prime_pages_to_sg(struct drm_device *dev,
  * OTHER DEALINGS IN THE SOFTWARE.
  */
 #include <drm/drm_crtc.h>
+#include <drm/drm_framebuffer.h>
 
 #if defined(drm_for_each_plane)
 #define nv_drm_for_each_plane(plane, dev) \
